easybuild/easyconfigs/b/BamM/BamM-1.7.3-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     GPL-v3.0
+#
+# Notes::
+##
+
+easyblock = 'PythonPackage'
+
+name = 'BamM'
+version = '1.7.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ecogenomics.github.io/BamM/'
+description = """ BamM is a c library, wrapped in python, that parses BAM files """
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/Ecogenomics/BamM/archive']
+sources = ['%(version)s.tar.gz']
+
+checksums = ['1a30a5014aa64aea23f12b82c8e474044af126c9e6ddb575530ab14b6ef765b8', 'sha256']
+
+buildopts = [
+    ('libtool', '2.4.6'),
```

Comment:
@robqiao could you explain what you're trying to do here? You've written this as though it's a dependency, so the build fails:
```
== 2017-07-28 10:28:26,238 run.py:132 DEBUG run_cmd: running cmd  /camp/apps/eb/software/Python/2.7.12-foss-2016b/bin/python setup.py build ('libtool', '2.4.6') (in /camp/apps/eb/review/build/BamM/1.7.3/foss-2016b-Python-2.7.12/BamM-1.7.3)
== 2017-07-28 10:28:26,238 run.py:157 INFO running cmd:  /camp/apps/eb/software/Python/2.7.12-foss-2016b/bin/python setup.py build ('libtool', '2.4.6')
== 2017-07-28 10:28:26,262 run.py:439 DEBUG cmd " /camp/apps/eb/software/Python/2.7.12-foss-2016b/bin/python setup.py build ('libtool', '2.4.6')" exited with exitcode 1 and output:
/bin/bash: -c: line 0: syntax error near unexpected token `('
/bin/bash: -c: line 0: ` /camp/apps/eb/software/Python/2.7.12-foss-2016b/bin/python setup.py build ('libtool', '2.4.6')'
```